base/doc/ltnews38.tex

```diff
@@ -266,12 +393,11 @@ \subsection{In the programming layer}
 \cs{fpeval}. For example, this could be used to create a new function
 \texttt{dinner}:
 \begin{verbatim}
-\documentclass{article}
-\begin{document}
 \ExplSyntaxOn
 \fp_new_variable:n{duck}
 \fp_new_function:n{dinner}
-\fp_set_function:nnn{dinner}{duck}{duck - 0.25 * duck}
+\fp_set_function:nnn{dinner}{duck}
+                    {duck - 0.25 * duck}
```

Comment:
I think this should be duck - 0.25 if 1/4 duck each or no `*` if multiplication intended  as multiplication is implicit

Reply:
what would I know about ducks :-) this is not my example, @josephwright ?

Reply:
Nope, @u-fischer

Reply:
As receipe for the dinner it should probably be simply `duck - 0.25` to that everyone get the same amount.

Reply:
oh no :duck:

Reply:
Yummy, either way!